lib/pardot/http.rb

```diff
@@ -13,10 +13,10 @@ def get object, path, params = {}, num_retries = 0
       raise Pardot::NetError.new(e)
     end
 
-    def post object, path, params = {}, num_retries = 0
+    def post object, path, params = {}, num_retries = 0, bodyParams = {}
```

Comment:
I think it might be clearer with a named param rather than just positional arguments.  I had a really hard time parsing the arguments to the `post` call in authentication.rb

Reply:
So none of these are my changes I've just merged them from upstream/master